lambdas/tests/eventbrite/handlers/producer.test.js

```diff
@@ -73,31 +73,30 @@ describe("Eventbrite Producer", function() {
       getFromWeb.mockImplementation(() => singleResponse());
     });
 
-    it("does not return an error and returns message", function(done) {
-      producer.produce(event, context, function(err, response) {
-        expect(err).toBe(null);
-        expect(response).toEqual({ message: ["path/to/new/file.json"] });
-        done();
-      });
-    });
-
     it("makes a request to the API", function(done) {
-      producer.produce(event, context, function() {
+      producer.produce(event, context, function(err) {
         expect(getFromWeb).toHaveBeenCalledTimes(1);
         expect(getFromWeb).toHaveBeenCalledWith(apiCallPage(1));
-        done();
+        done(err);
```

Comment:
`done` allows passing the first argument as an error to cause the test to fail (which allows it to be glued in to the standard callback pattern) - in this case, we expect `err` will be `null` here so the test won't fail, but if `err` was ever set to an error, the test would fail as there would have been an issue within the `produce` function.